tests/models/blip_2/test_modeling_blip_2.py

```diff
@@ -1698,9 +1706,19 @@ def test_inference_t5(self):
         predictions = model.generate(**inputs)
         generated_text = processor.batch_decode(predictions, skip_special_tokens=True)[0].strip()
 
+        expectations = Expectations(
```

Comment:
Here and blow in test file: just to change to format to use `Expectations(...)` so we can easily add values for `A10` and `XPU`